code/core/src/core-server/manifest.ts

```diff
@@ -0,0 +1,381 @@
+  const entries = Object.entries(manifest?.components ?? {}).sort((a, b) =>
+    (a[1].name || a[0]).localeCompare(b[1].name || b[0])
+  );
+
+  const analyses = entries.map(([, c]) => analyzeComponent(c));
+  const totals = {
+    components: entries.length,
+    componentsWithError: analyses.filter((a) => a.hasComponentError).length,
+    componentsWithWarnings: analyses.filter((a) => a.hasWarns).length,
+    examples: analyses.reduce((sum, a) => sum + a.totalExamples, 0),
+    exampleErrors: analyses.reduce((sum, a) => sum + a.exampleErrors, 0),
+  };
+
+  // Top filters (clickable), no <b> tags; 1px active ring lives in CSS via :target
+  const allPill = `<a class="filter-pill all" data-k="all" href="#filter-all">All</a>`;
+  const compErrorsPill =
+    totals.componentsWithError > 0
+      ? `<a class="filter-pill err" data-k="errors" href="#filter-errors">${totals.componentsWithError}/${totals.components} component ${plural(totals.componentsWithError, 'error')}</a>`
+      : `<span class="filter-pill ok" aria-disabled="true">${totals.components} components ok</span>`;
+  const compWarningsPill =
+    totals.componentsWithWarnings > 0
+      ? `<a class="filter-pill warn" data-k="warnings" href="#filter-warnings">${totals.componentsWithWarnings}/${totals.components} component ${plural(totals.componentsWithWarnings, 'warning')}</a>`
+      : '';
+  const examplesPill =
+    totals.exampleErrors > 0
+      ? `<a class="filter-pill err" data-k="example-errors" href="#filter-example-errors">${totals.exampleErrors}/${totals.examples} example errors</a>`
+      : `<span class="filter-pill ok" aria-disabled="true">${totals.examples} examples ok</span>`;
+
+  const grid = entries.map(([key, c], idx) => renderComponentCard(key, c, `${idx}`)).join('');
+
+  const errorGroups = Object.entries(
+    groupBy(
+      entries.map(([, it]) => it).filter((it) => it.error),
+      (manifest) => manifest.error?.name ?? 'Error'
+    )
+  );
+
+  const errorGroupsHTML = errorGroups
+    .map(([error, grouped]) => {
+      const id = error.toLowerCase().replace(/[^a-z0-9]+/g, '-');
+      const headerText = `${esc(error)}`;
+      const cards = grouped
+        .map((manifest, id) => renderComponentCard(manifest.id, manifest, `error-${id}`))
+        .join('');
+      return `
+        <section class="group">
+          <input id="${id}-toggle" class="group-tg" type="checkbox" hidden />
+          <label for="${id}-toggle" class="group-header">
+            <span class="caret">▸</span>
+            <span class="group-title">${headerText}</span>
+            <span class="group-count">${grouped.length}</span>
+          </label>
+          <div class="group-cards">${cards}</div>
+        </section>
+      `;
+    })
+    .join('');
+
+  return `<!doctype html>
+<html lang="en">
+<head>
+<meta charset="utf-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1" />
+<title>Components Manifest</title>
+<style>
```

Comment:
at least this style tag needs to be passed through prettier once. just paste it into a css file, format with prettier, then put it in here again.